pkg/setup/config.go

```diff
@@ -38,8 +39,11 @@ func InitClients(vConfig *config.VirtualClusterConfig) error {
 	}
 
 	// get workload target namespace
-	if vConfig.Experimental.MultiNamespaceMode.Enabled {
-		translate.Default = translate.NewMultiNamespaceTranslator(vConfig.WorkloadNamespace)
+	if vConfig.Sync.ToHost.Namespaces.Enabled {
+		translate.Default, err = pro.GetWithSyncedNamespacesTranslator(vConfig.WorkloadNamespace, vConfig.Sync.ToHost.Namespaces.Mappings)
```

Comment:
I don't understand this, does that mean we don't use translate.NewMultiNamespaceTranslator anymore at all?

Reply:
yeah exactly. I didn't clean it up yet but current behaviour for namespace syncing is:
- use only new namespace translator
- allow only operations on namespaces defined in mappings
- for special cases fallback to singleNamespace logic